cypress/e2e/data-import/e2e/match-on-vrn-and-update-items.cy.js

```diff
@@ -36,6 +37,7 @@ describe('ui-data-import', () => {
     quantityPhysical: '1',
     createInventory: 'Instance, holdings, item'
   };
+  const itemBarcode = 'xyzt124245271818912626262';
```

Comment:
why don't you use here the random barcode?

Reply:
Done